study.md

```diff
@@ -147,5 +159,5 @@ using `hank` that equals "My name is Hank Hill and I'm a propane and propane
 accesories salesman that enjoys drinking beer.".
 
 ```javascript
-// your answer here
+`My name is ${hank.givenName} ${hank.familyName} and I'm a ${hank.hobbies(1)} salesman that enjoys ${hank.hobbies(0)}.`
```

Comment:
Array indices need to be denoted by surrounding brackets not parentheses.  As it is, `hank.hobbies(1)` will attempt to call the value of `hank.hobbies` as if it were a function and cause an error.